api/validator.js

```diff
@@ -13,7 +13,7 @@ const SCHEMA_JOB_COMMAND = Joi.object()
         name: Joi.string(),
         command: Joi.string()
     })
-    .unknown(false)
```

Comment:
should we just add `locked` as optional here, instead of allowing `unknown` ?

Reply:
added